discordbot/economy/currencies.py

```diff
@@ -5,34 +5,61 @@
 
 
 async def currencies_command(ctx, arg):
-    # Help
-    if arg == "-h" or arg == "help":
-        help_txt = "Currencies [Source: Wall St. Journal]\n"
-        embed = discord.Embed(
-            title="Economy: [WSJ] Currencies HELP",
-            description=help_txt,
-            colour=cfg.COLOR,
-        )
-        embed.set_author(
-            name=cfg.AUTHOR_NAME,
-            icon_url=cfg.AUTHOR_ICON_URL,
-        )
+    try:
+        # Debug
+        if cfg.DEBUG:
+            print("-- STARTED COMMAND: !stocks.economy.currencies " + arg + " --")

```

Comment:
Can I suggest using python3 print convention?

Like
```
print(f"-- STARTED COMMAND: !stocks.economy.currencies {arg} --")
```

Reply:
Also, we can just have: 
```
print(f"!stocks.economy.currencies {arg}")
```

And then if there's a problem we know it will be associated with this

Reply:
Sounds good! I forgot the python 3 convention.

Reply:
Fixed it!